src/replication.c

```diff
@@ -873,13 +925,21 @@ update_replica_entry(spec_t *spec, replica_t *replica, int iofd)
 	}
 
 	MTX_LOCK(&spec->rq_mtx);
-	if(is_rf_replicas_connected(spec, replica)) {
+	if (is_rf_replicas_connected(spec, replica)) {
 		MTX_UNLOCK(&spec->rq_mtx);
 		ISTGT_ERRLOG("failed to verify replica(ip:%s port:%d "
 		    "guid:%lu)\n", replica->ip, replica->port,
 		    replica->zvol_guid);
 		goto replica_error;
 	}
+
+	if (!is_replica_familiar(spec, replica)) {
```

Comment:
'is_replica_new_or_familar' may be good?

Reply:
ok